src/sdk-reference/js/6/getting-started/node-js/index.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: full.html.hbs
+algolia: true
+title: Node.js
+description: Getting started with Kuzzle and Node.js
+---
+
+# Getting Started with Kuzzle and Node.js
+
+This tutorial explains you how to use **Kuzzle** with **Node.js** and the **Javascript SDK**.  
+It will walk you through creating scripts that can **store** documents in Kuzzle and subscribe to **notifications** about document creations.
+
+You are going to write an application that **stores** documents in Kuzzle Server and subscribe to **real time notifications** for each created document.
+
+To follow this tutorial, you must have a Kuzzle Server up and running. Follow these instructions if this is not already the case: [Running Kuzzle]({{ site_base_path }}guide/1/getting-started/running-kuzzle).
+
+## Explore the SDK
+
+It's time to get started with the [Kuzzle Javascript SDK]({{ site_base_path }}sdk-reference/js/6). This section, explains you how to store a document and subscribe to notifications in Kuzzle using the Javascript SDK.
+
+Before proceeding, please make sure your system has **Node.js** version 8 or higher (<a href="https://nodejs.org/en/download/">instructions here</a>) installed.
+
+## Prepare your environment
+
+Create your playground directory and install the Javascript SDK from the command line using npm:
+
+```sh
+mkdir "kuzzle-playground"
+cd "kuzzle-playground"
+npm install git://github.com/kuzzleio/sdk-javascript.git#6-beta
+```
+
+<div class="alert alert-info">
+If you are performing a clean install you might get some `UNMET PEER DEPENDENCY` warnings, these are safe to ignore as they refer to optional dependencies.
+</div>
+
+Then, create an `init.js` file and start by adding the code below.  
+This loads the SDK and connects it to a Kuzzle instance using WebSocket.  
+
+[snippet=load-sdk]
+
+<div class="alert alert-info">
+Replace 'kuzzle' which is the Kuzzle backend hostname with 'localhost' or the hostname where your Kuzzle backend is running.
```

Comment:
https://confluence.kaliop.net/display/LABS/2018-02-21+Meeting+notes++-+Kuzzle+products+naming
I think we should use Kuzzle Server instead of Kuzzle backend